XSLT/mtsucoll17dctomods.xsl

```diff
@@ -0,0 +1,211 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+    xmlns:oai_dc='http://www.openarchives.org/OAI/2.0/oai_dc/' xmlns:dc="http://purl.org/dc/elements/1.1/" 
+    xsi:schemaLocation="http://www.openarchives.org/OAI/2.0/"
+    version="2.0" xmlns="http://www.loc.gov/mods/v3" xmlns:xlink="http://www.w3.org/1999/xlink" exclude-result-prefixes="dc oai_dc">
+    <xsl:output omit-xml-declaration="yes" method="xml" encoding="UTF-8" indent="yes"/>
+    
+    <!-- output settings -->
+    <xsl:output encoding="UTF-8" method="xml" omit-xml-declaration="yes" indent="yes"/>
+    <xsl:strip-space elements="*"/>
+    
+    <!-- identity transform -->
+    <xsl:template match="@* | node()">
+        <xsl:copy>
+            <xsl:apply-templates select="@* | node()"/>
+        </xsl:copy>
+    </xsl:template>
+    
+    <!-- normalize all the text! -->
+    <xsl:template match="text()">
+        <xsl:value-of select="normalize-space(.)"/>
+    </xsl:template>
+    
+    <!-- match metadata -->
+    <xsl:template match="oai_dc:dc">
+        
+        <!-- match the document root and return a MODS record -->
+        <mods xmlns="http://www.loc.gov/mods/v3" version="3.5"
+            xmlns:xlink="http://www.w3.org/1999/xlink"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="http://www.loc.gov/mods/v3 http://www.loc.gov/standards/mods/v3/mods-3-5.xsd">
+            
+            <!-- title -->
+            <xsl:apply-templates select="dc:title"/> 
+            
+            <!-- identifier -->
+            <xsl:apply-templates select="dc:identifier"/>
+            
+            <!-- abstract -->
+            <xsl:apply-templates select="dc:description"/>
+            
+            <!-- creator -->
+            <xsl:apply-templates select="dc:creator"/>
+            
+            <!-- contributor -->
+            <xsl:apply-templates select="dc:contributor"/>
+            
+            <!-- date -->
+            <xsl:apply-templates select="dc:date"/>
+            
+            <!-- geographic> -->
+            <xsl:apply-templates select="dc:coverage"/>
+            
+            <!-- subject(s) -->
+            <xsl:apply-templates select="dc:subject"/>
+            
+            <!-- note -->
+            <xsl:apply-templates select="dc:source"/>
+            
+            <!-- typeOfResource -->
+            <xsl:apply-templates select="dc:type"/>
+            
+            <!-- recordContentSource -->
+            <recordInfo>
+                <recordContentSource>Digital Initiatives, James E. Walker Library, Middle Tennessee State University</recordContentSource>
+            </recordInfo>
+            
+            <!-- accessCondition -->
+            <xsl:apply-templates select="dc:rights"/>
```

Comment:
@mlhale7 this approach here is where you're having problems with `accessCondition`. The way it's written, you're only going to transform dc:rights to an accessCondition if dc:rights exists because you're running apply templates on the presence of that node.  See how you're doing `<recordInfo/>` above?  You basically need something like that.  There are many approaches, but what about:

```
<!-- accessCondition -->
            <xsl:choose>
                <xsl:when test="dc:rights">
                    <xsl:apply-templates select="dc:rights"/>
                </xsl:when>
                <xsl:otherwise>
                    <accessCondition type="use and reproduction" xlink:href="http://rightsstatements.org/vocab/CNE/1.0/">Copyright Not Evaluated</accessCondition>
                </xsl:otherwise>
            </xsl:choose>
```